packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -293,6 +294,10 @@ export default class TradeStore extends BaseStore {
     // Mobile
     is_trade_params_expanded = true;
 
+    debouncedSendTradeParamsAnalytics = debounce((payload: TEvents['ce_contracts_set_up_form']) => {
```

Comment:
Should we clean it somewhere? `debouncedSendTradeParamsAnalytics.cancel();`
Please, ignore if not relevant

Reply:
@kate-deriv no, we don't :) it's created just once, that's why it doesn't require cleaning before next execution. and it's located in the store, not inside a react component :) It's working the same way as `debouncedProposal`.

Reply:
Thank you for explanation